examples/recipes/multiple-data-sources/cube.js

```diff
@@ -0,0 +1,28 @@
+const PostgresDriver = require('@cubejs-backend/postgres-driver');
+
+module.exports = {
+  // Provides distinct identifiers for each tenant which is used as caching key
+  contextToAppId: ({ securityContext }) =>
+    `CUBEJS_APP_${securityContext.tenant}`,
+
+  // Selects the database connection configuration based on the tenant name
+  driverFactory: ({ securityContext } = {}) => {
+    if (securityContext.tenant === 'Avocado Inc') {
+      return new PostgresDriver({
+        database: 'localDB',
+        host: 'postgres',
+        user: 'postgres',
+        password: 'example',
+        port: '5432',
+      });
+    } else {
```

Comment:
I think it's not a best practice when we allow an unknown tenant to access this tenant's schema. I think we should check for the other tenant's name as well—and throw an exception, failing the query, if the tenant is unset or unknown. You can also show this case with an unset or unknown tenant in the "Query" section.

Reply:
Make sense. I added the tenant's name check and updated the "Query" and "Result" sections.